### PR TITLE
🏁 Blueprint EOL

### PR DESCRIPTION
**Pull request by chapulina**
*Do not merge until January 1st 2021.*

Library versions EOLing with Blueprint:

* ign-fuel-tools3
* ign-gazebo2
* ign-gui2
* ign-launch1
* ign-msgs4
* ign-physics1
* ign-rendering2
* ign-sensors2
* ign-transport7
* ign-tools0
* sdf8

I also removed some versions that had long been EOL, because they're not used by any Ignition or classic versions:

* `ign-cmake1`
* `ign-common2`
* `ign-fuel-tools2`
* `ign-math5`
* `ign-msgs2`
* `ign-transport5`

Also, not sure why it wasn't there, but added `ign-transport2`, which is used by Gazebo 7 (to EOL in 1 month)

### CODE REVIEW

**Comment by scpeters** [comment]
I think we can disable CI for sdformat8 as well:

~~~
diff --git a/jenkins-scripts/dsl/sdformat.dsl b/jenkins-scripts/dsl/sdformat.dsl
index e8a81ed0..f6681663 100644
--- a/jenkins-scripts/dsl/sdformat.dsl
+++ b/jenkins-scripts/dsl/sdformat.dsl
@@ -1,8 +1,8 @@
 import _configs_.*
 import javaposse.jobdsl.dsl.Job
 
-def sdformat_supported_branches = [ 'sdformat4', 'sdformat6', 'sdformat8' , 'sdformat9', 'sdformat10' ]
-def sdformat_gz11_branches = [ 'sdformat8', 'sdformat9', 'sdformat10', 'master' ]
+def sdformat_supported_branches = [ 'sdformat4', 'sdformat6', 'sdformat9', 'sdformat10' ]
+def sdformat_gz11_branches = [ 'sdformat9', 'sdformat10', 'master' ]
 // nightly and prereleases
 def extra_sdformat_debbuilder = [ ]
 
~~~

**Comment by chapulina** [comment]
> I think we can disable CI for sdformat8 as well

225e643

**Review by scpeters — APPROVED** [review]

**Review by j-rivero — COMMENTED** [review]
I made a dependencies_archive clean up. It probably needs a bit more to be fully updated but the changes should reflect modifications of this PR. I would like a quick double check on that commit and a bit of testing since changes in that file potentially affect many different builds.

File: jenkins-scripts/lib/dependencies_archive.sh [hunk 1]
```diff
@@ -784,7 +784,7 @@ SUBT_DEPENDENCIES="wget \\
                    curl \\
                    git  \\
                    ${ROS_CATKIN_BASE} \\
-                   ignition-blueprint  \\
+                   ignition-dome  \\
```
[the remark this answers is not in the document]
Comment (scpeters) [reply jenkins-scripts/lib/dependencies_archive.sh:718]: edifice is not ready yet

**Review by scpeters — APPROVED** [review]

**Comment by chapulina** [comment]
>  I would like a quick double check on that commit 

It looks good to me :+1: 

Merging

**Pull request merged**